src/main/resources/config/restricted-fields.json

```diff
@@ -0,0 +1,59 @@
+{
+  "SMC" : {
```

Comment:
I think this 'soft link' is fine looking through the document removes a load of UUIDs too